solana/rpc/_utils/encoding.py

```diff
@@ -26,7 +26,7 @@ def _json_list_errors(self, iterable: Iterable[Any]) -> Iterable[str]:
             try:
```

Comment:
Can we make these changes in a separate PR?

Reply:
(Un)done